From 9d9d0c1fd3d4b6d38c3aba90d175252daba860ca Mon Sep 17 00:00:00 2001
From: Bhavya Kapoor <b-kapoor@ti.com>
Date: Mon, 26 Feb 2024 15:09:55 +0530
Subject: [PATCH 19/27] arm64: dts: ti: k3-j722s-evm: Enable UHS modes for
 MMCSD

Remove no-1-8-v property to enable support for ultra
high speed modes for MMCSD in J722S SoC.

Signed-off-by: Bhavya Kapoor <b-kapoor@ti.com>
---
 arch/arm64/boot/dts/ti/k3-j722s-evm.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/ti/k3-j722s-evm.dts b/arch/arm64/boot/dts/ti/k3-j722s-evm.dts
index 11f99e1ee226..dc27ae9a078c 100644
--- a/arch/arm64/boot/dts/ti/k3-j722s-evm.dts
+++ b/arch/arm64/boot/dts/ti/k3-j722s-evm.dts
@@ -578,7 +578,6 @@ &sdhci1 {
 	pinctrl-0 = <&main_mmc1_pins_default>;
 	ti,driver-strength-ohm = <50>;
 	disable-wp;
-	no-1-8-v;
 	bootph-all;
 };
 
-- 
2.39.2

